pdc_client/plugins/component.py

```diff
@@ -64,6 +65,13 @@ def list_global_components(self, args):
                       global_component['id'],
                       global_component['name'])
 
+    def get_component_id(self, args):
```

Comment:
Is it better name the function as _get_component_id? It's more like a class private function.